src/other_functions.jl

```diff
@@ -182,7 +183,7 @@ Inexact equality comparison between polynomials: returns `true` if
 `norm(x-y,1) <= atol + rtol*max(norm(x,1), norm(y,1))`, where `x` and `y` are
 polynomials. For more details, see [`Base.isapprox`](@ref).
 """
-function isapprox(x::T, y::S; rtol::Real=rtoldefault(x,y), atol::Real=0,
+function isapprox(x::T, y::S; rtol::Real=rtoldefault(x,y), atol::Real=0.0,
```

Comment:
I don't see the need for this change (from `0` to `0.0`).

Reply:
`atol` is usually some sort of Float; I though like this I avoided a conversion. I am leaving it as it is, buty it may be changed eventually.